packages/runtimes/hmr/src/loaders/hmr-runtime.js

```diff
@@ -93,6 +99,25 @@ if ((!parent || !parent.isParcelRequire) && typeof WebSocket !== 'undefined') {
     protocol + '://' + hostname + (port ? ':' + port : '') + '/',
   );
 
+  var globalObject =
+    typeof globalThis !== 'undefined'
+      ? globalThis
+      : typeof self !== 'undefined'
+      ? self
+      : typeof window !== 'undefined'
+      ? window
+      : typeof global !== 'undefined'
+      ? global
+      : {};
```

Comment:
Why was this necessary? `global` should be injected by parcel when needed...

Reply:
It wasn't for some reason. I don't think `global` is actually a property that exists in the DOM anyway...just in Node.js

Reply:
Parcel should be polyfilling it https://parceljs.org/features/node-emulation/#shimming-builtin-node-globals

Reply:
I tested and it seems to be working, but maybe I missed something